metadata.md

```diff
@@ -0,0 +1,98 @@
+# Metadata Schema - Version 1.0
```

Comment:
Do we want to start with `1.0`? I tend to number my own projects starting with `0.1`, and move to `1.0` when I consider it stable.

Reply:
I do the same for code.  I think the common convention with standards / specifications is to start with 1.0, but I don't have a strong opinion on it here.

Reply:
I was unaware, but I checked a couple specs, e.g. TileJSON, and that does indeed start with 1.0.

Reply:
It might still be good to call it "in beta", or something like that, for now. While gathering feedback in the short term, we might still want to change some things?

Reply:
After I considered versioning further, I think some of this comes down to implementation readiness and compatibility promises.

1.0 implies it is implementation ready, and that we're willing to make compatibility promises not to make breaking changes lightly (and in a semantic-version controlled manner when we do).

I don't think we're ready for that level yet.  I think in geopandas we're experimenting with this first pass, so we'll need to have warnings about stability of the spec / implementation over there.

Here, we should instead note that the purpose of this is a first daft to outline the concept and gather feedback from the community.  We'll need to add warnings about stability here too; people should be encouraged to start implementing this, so that we can learn from those experiments.  But people should be cautious about using this for production just yet.

So - I think we should mark this as `0.1.0` to indicate those things.  Any issue with that?

Reply:
I agree with your conclusions, no issue with that.